contrib/storage-hive/core/src/test/java/org/apache/drill/exec/hive/TestHiveStorage.java

```diff
@@ -74,10 +73,8 @@ public void testNativeScanWhenNoColumnIsRead() throws Exception {
           .baselineValues(200l)
           .go();
     } finally {
-      final OperatorFixture.TestOptionSet testOptionSet = new OperatorFixture.TestOptionSet();
-      test(String.format("alter session set `%s` = %s",
-          ExecConstants.HIVE_OPTIMIZE_SCAN_WITH_NATIVE_READERS,
-          Boolean.toString(testOptionSet.getDefault(ExecConstants.HIVE_OPTIMIZE_SCAN_WITH_NATIVE_READERS).bool_val)));
+      test(String.format("alter session reset `%s`",
```

Comment:
Minor nitpick. The String.format is not necessary here. BaseTestQuery has a test method that does the String.format for you.